projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-file.tsx

```diff
@@ -251,6 +259,7 @@ export class DotBinaryFileComponent {
                 </dot-label>
                 {getTagHint(this.hint)}
                 {getTagError(this.shouldShowErrorMessage(), this.getErrorMessage())}
+                {this.errorMessage ? <div class="dot-binary__error-message">{this.errorMessage}</div> : ''}
```

Comment:
Why don't we use here: `dot-form-error-message` component so we don't have to add any extra styling